src/Maker/MakeCommand.php

```diff
@@ -58,10 +63,22 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
     {
         $command
             ->addArgument('name', InputArgument::OPTIONAL, \sprintf('Choose a command name (e.g. <fg=yellow>app:%s</>)', Str::asCommand(Str::getRandomTerm())))
+            ->addOption('invokable', null, InputOption::VALUE_NEGATABLE, 'Generate an invokable command instead of a command with the old structure (configure and execute method)?')
```

Comment:
Could be enough I think
```suggestion
            ->addOption('invokable', null, InputOption::VALUE_NEGATABLE, 'Generate an invokable command (using PHP attributes) starting from Symfony 7.3?')
```